utils/update_checkout/update_checkout/update_checkout.py

```diff
@@ -100,20 +115,25 @@ def get_branch_for_repo(config, repo_name, scheme_name, scheme_map,
     cross_repo = False
     repo_branch = scheme_name
     if scheme_map:
-        scheme_branch = scheme_map[repo_name]
-        repo_branch = scheme_branch
-        remote_repo_id = config['repos'][repo_name]['remote']['id']
-        if remote_repo_id in cross_repos_pr:
-            cross_repo = True
-            pr_id = cross_repos_pr[remote_repo_id]
-            repo_branch = "ci_pr_{0}".format(pr_id)
-            shell.run(["git", "checkout", scheme_branch],
-                      echo=True)
-            shell.capture(["git", "branch", "-D", repo_branch],
-                          echo=True, allow_non_zero_exit=True)
-            shell.run(["git", "fetch", "origin",
-                       "pull/{0}/merge:{1}"
-                       .format(pr_id, repo_branch), "--tags"], echo=True)
+        try:
+            scheme_branch = scheme_map[repo_name]
+            repo_branch = scheme_branch
+            remote_repo_id = config['repos'][repo_name]['remote']['id']
+            if remote_repo_id in cross_repos_pr:
+                cross_repo = True
+                pr_id = cross_repos_pr[remote_repo_id]
+                repo_branch = "ci_pr_{0}".format(pr_id)
+                shell.run(["git", "checkout", scheme_branch],
+                          echo=True)
+                shell.capture(["git", "branch", "-D", repo_branch],
+                              echo=True, allow_non_zero_exit=True)
+                shell.run(["git", "fetch", "origin",
+                           "pull/{0}/merge:{1}"
+                           .format(pr_id, repo_branch), "--tags"], echo=True)
+        except KeyError as e:
+            print(f"Failed to look up {repo_name} in scheme", file=sys.stderr)
+            raise e
+            exit(1)
```

Comment:
If you want to exit, this should be `sys.exit` instead of `exit` (the latter is designed for interactive use).
But, should this just be removed since it's right after that `raise`?